university-of-southern-queensland-harvard-agps.csl

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" et-al-min="4" et-al-use-first="1" initialize-with-hyphen="false" demote-non-dropping-particle="never" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>University of Southern Queensland - Harvard AGPS</title>
+    <title-short>HarvardAGPS-UniSQ</title-short>
+    <id>http://www.zotero.org/styles/university-of-southern-queensland-harvard-agps</id>
+    <link href="http://www.zotero.org/styles/university-of-southern-queensland-harvard-agps" rel="self"/>
+    <link href="http://www.zotero.org/styles/university-of-south-australia-2017-harvard" rel="template"/>
+    <link href="https://usq.pressbooks.pub/harvardagps/" rel="documentation"/>
+    <author>
+      <name>Dale Hitchenor</name>
+      <email>dhitchenor@disroot.org</email>
+      <uri>https://dale.hitchenor.com</uri>
+    </author>
+    <category citation-format="author-date"/>
+    <summary>University of Southern Queensland citation style, based on the following manual: Snooks &amp; Co. 2002, Style manual for authors, editors and printers, 6th edition John Wiley &amp; Sons, [Milton, Qld].</summary>
+    <updated>2025-10-03T04:42:29+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
```

Comment:
Most of these term overrides you've introduced are already included in our standard en-GB locale. Please delete all duplications: https://github.com/citation-style-language/locales/blob/master/locales-en-GB.xml